src/components/previewers/ImagePreviewer.jsx

```
@@ -49,108 +61,144 @@ const ImagePreviewer = ({
 
-            {/* Description Section */}
-            <div className="bg-white p-4 w-full dark:bg-dark_bg_2 md:w-1/3 overflow-y-auto">
-              <p className="text-gray-700 dark:text-dark_text_1">
-                {description}
-              </p>
+                  {!isEmoji && (
+                    <button
+                      onClick={handleCopy}
+                      className={`action-button hover:${textColor}`}
+                      aria-label="Copy Link"
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)